ansible/host_vars/events.proteus.test.ooni.io/vars.yml

```diff
@@ -0,0 +1,2 @@
+---
+adm_logins: [ art, darkk, sbs, nuke ]
```

Comment:
There is no `weasel` login here. Does it mean that it should be dropped from [`all`](https://github.com/ooni/sysadmin/blob/master/ansible/group_vars/all/vars.yml#L105) as well?